org.sugarlabs.MazeActivity.json

```diff
@@ -0,0 +1,242 @@
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://www.github.com/TelepathyIM/telepathy-glib.git",
+                    "tag": "telepathy-glib-0.24.1",
+                    "commit": "c834ef1a50a97cbfae7a0d560ff806fd5aa7ffe8"
+                },
+                {
+                    "type": "patch",
+                    "path": "telepathy-glib-uniquify.patch"
+                }
+            ]
+        },
+        {
+            "name": "sugar-toolkit-gtk3",
+            "no-parallel-make": true,
+            "config-opts": [
+                "--with-python3"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://download.sugarlabs.org/sources/sucrose/glucose/sugar-toolkit-gtk3/sugar-toolkit-gtk3-0.116.tar.xz",
+                    "sha256": "745a360577efceb10f57ca4cbad5f9ec61f7ed6cc381eeb72422bbd7455b770b"
+                }
+            ]
+        },
+        {
+            "name": "sugarapp",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --no-index --find-links ${PWD} --install-option=\"--prefix=${FLATPAK_DEST}\" $PWD"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/tchx84/sugarapp.git",
+                    "tag": "v1.10",
+                    "commit": "54b0efdc0bbef60d48685aa2a1559a3fb8f2ddf5"
+                }
+            ]
+        },
+        {
+            "name": "maze",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=${FLATPAK_DEST} --skip-install-desktop-file --skip-install-mime"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/sugarlabs/maze-activity.git",
+                    "commit": "01fadbac72c003708a0a36d4eefacc5c5124ffbe"
+                },
+                {
+                    "type": "patch",
+                    "path": "maze-port.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "maze-monitors.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "maze-info.patch"
+                }
+            ],
+            "post-install": [
+                "sugarapp-gen-mimetypes activity/activity.info mimetypes",
+                "sugarapp-gen-appdata activity/activity.info appdata",
+                "sugarapp-gen-desktop activity/activity.info desktop --mimetypes mimetypes",
+                "install -D mimetypes /app/share/mime/packages/org.sugarlabs.MazeActivity.xml",
+                "install -D mimetypes /app/share/sugar/activities/Maze.activity/activity/mimetypes.xml",
```

Comment:
No need to install `mimetypes` (which shouldn't exists).

Reply:
9656eaa